plutus-tx-plugin/src/PlutusTx/Compiler/Builtins.hs

```diff
@@ -725,6 +727,8 @@ defineBuiltinTerms = do
           -- Value
           PLC.InsertCoin -> defineBuiltinInl 'Builtins.insertCoin
           PLC.UnionValue -> defineBuiltinInl 'Builtins.unionValue
+          PLC.ValueData -> defineBuiltinInl 'Builtins.mkValue
```

Comment:
Is there a reason to have different names `ValueData` and `mkValue` ?

Reply:
Not sure, but that's how other Data primitives are named (mkI, mkB etc.)